### PR TITLE
MANTA-4387 [mantav2] update Moray to use artedi v2 fixed buckets histograms

### PR DESCRIPTION
**Pull request by khalfella**

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -153,6 +153,7 @@ function MorayServer(options) {
         help: 'The node-fast CRC compatibilty mode of the Fast server'
     }).set(crc_mode);
 
+    collector.FIXED_BUCKETS = true;
```
[the remark this answers is not in the document]
Comment (khalfella) [reply lib/server.js]: Removed the line that sets `FIXED_BUCKETS`

**Comment by khalfella** [comment]
The build has completed successfully, but for jenkins failed to update the PR in github. Here is the error message from jenkins build log.

```
Could not update commit status. Message: Server returned HTTP response code: -1, message: 'null' for URL: https://api.github.com/repos/joyent/moray

Finished: SUCCESS
```

File: lib/server.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (khalfella) [reply lib/server.js]: ping @joshwilsdon, @trentm.

**Comment by trentm** [comment]
@khalfella FYI there is a "re-request review" button in the "Reviewers" section if you want to make sure that a request for review is shown on that reviewer's dashboard after you've made a change to a PR.

**Comment by khalfella** [comment]
> @khalfella FYI there is a "re-request review" button in the "Reviewers" section if you want to make sure that a request for review is shown on that reviewer's dashboard after you've made a change to a PR.

Thank you. I didn't about "re-request review". Now I know about it, I will start using it in the future.

**Pull request merged**